internal/npm_install/generate_build_file.js

```diff
@@ -966,7 +967,11 @@ nodejs_binary(
  * Given a pkg, return the skylark aliases for the package.
  */
 function printPackageAliases(pkg) {
-  return `
+
+  const pkgJson = path.posix.join(pkg._dir, 'package.json');
```

Comment:
You shouldn't need to re-read the packageJson off the disk, `pkg` already has all the information you need

Reply:
You shouldn't need these changes anymore.